packages/components/src/custom-select-control/README.md

```diff
@@ -114,14 +112,6 @@ Can be used to externally control the value of the control, like in the `MyContr
 -   Type: `Object`
 -   Required: No
 
-#### __nextUnconstrainedWidth
```

Comment:
Should we also remove the corresponding prop from the legacy types:

https://github.com/WordPress/gutenberg/blob/5939c41670df4f2caee40c24806d3be20a09dab1/packages/components/src/custom-select-control-v2/types.ts#L171

Reply:
Based on our discussion in #55401, we're going to keep it in the types indefinitely so consumer code can continue to compile, and to prevent future prop name collisions.

Reply:
Sounds good 👍